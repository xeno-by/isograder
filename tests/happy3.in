* Test [/10]
-1: foo
** Sub1 [/3]
-1: foo
** Sub2 [/3]
-1: foo
** Sub3 [/4]
-1: foo
*** Sub31 [/2]
-1: foo
*** Sub32 [/2]
-1: foo
