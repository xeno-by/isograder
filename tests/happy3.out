EXIT CODE:
0

STDOUT:
* Test [4/10]
-1: foo
** Sub1 [2/3]
-1: foo
** Sub2 [2/3]
-1: foo
** Sub3 [1/4]
-1: foo
*** Sub31 [1/2]
-1: foo
*** Sub32 [1/2]
-1: foo

STDERR:

